glide-core/src/errors.rs

```diff
@@ -0,0 +1,29 @@
+use redis::RedisError;
+
+pub enum RequestErrorType {
+    Unspecified,
+    ExecAbort,
+    Timeout,
+    Disconnect,
```

Comment:
`ClosingError` is missing

Reply:
when would an FFI client return a closing error?

Reply:
Is closing error applicable for UDS clients only?

Reply:
ATM yes. Only UDS clients have a shared resource that might be reach an unusable state.